content/using/operations/hosting.md

```diff
@@ -17,12 +17,9 @@ This guide uses Digital Ocean as the cloud provider, but others can be used.
 ## Create a Digital Ocean droplet
  - Create an account on [Digital Ocean][Digital Ocean].
  - Create a droplet with the following settings:
- - **Image**: Ubuntu 18.04 x64
- - **Plan**: Standard
- - **Size**: 
-   - **Minimum**: $10 (2GB/1CPU) 
-   - **Recommended**: $15 (3GB/1CPU) 
-   - **Bonus**: $20 (4GB/2CPU)
+ - **Image**: Ubuntu 20.04 x64
+ - **Plan**: Basic
+ - **Size**: $20 (4GB/2CPU)
```

Comment:
The exact plan doesn't matter. Just specify how much RAM you'll want.

Reply:
I've been suggesting 4GB because the swap stuff of 2GB and below seems like a pain, is that what I should do here? or say at least 3GB?

2GB with a comment about swap?

Reply:
I just set it to 4GB